### PR TITLE
EVG-13594 add service flag for github repo poller

### PR DESCRIPTION
**Pull request by ablack12**
I'll make another ticket to remove the poller altogether if we haven't heard of any issues in say a month or two.

### CODE REVIEW

File: units/crons.go [hunk 1]
```diff
@@ -42,6 +42,9 @@ func PopulateCatchupJobs() amboy.QueueOperation {
 			})
 			return nil
 		}
+		if flags.RepoPollerDisabled {
+			return nil
```
Comment (bsamek) [inline units/crons.go:46]: Could you add a log message here, similar to the one above for `flags.RepotrackerDisabled`?

File: service/templates/admin.html [hunk 2]
```diff
@@ -350,17 +361,6 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 													</md-radio-group>
 												</td>
 											</tr>
-											<tr>
-												<td>Projects Track Multiple Branches</td>
```
Comment (bsamek) [inline service/templates/admin.html:354]: Why are you removing this?

**Review by bsamek — CHANGES_REQUESTED** [review]

File: units/crons.go (hunk 1 above)
Comment (ablack12) [reply units/crons.go:46]: I thought about that, but we do intend to disable this right away and keep it disabled for a while, so it's going to be logging pretty frequently, whereas we'd only disable the repotracker overall in a crisis. I'm not sure this log would be valuable.

File: service/templates/admin.html (hunk 2 above)
Comment (ablack12) [reply service/templates/admin.html:354]: Sorry this is unrelated, just dead code I noticed while testing; should've been removed earlier.

**Review by bsamek — APPROVED** [review]

**Pull request merged**